simulators/active_surface.py

```diff
@@ -262,7 +250,7 @@ def set_stop_io(self, param):
     def set_positioning_io(self, param):
         binary_string = bin(param)[2:].zfill(8)
 
-        # binary_string[0 - 1] is currently unused
+        # binary_string[0:1] is currently unused
```

Comment:
This is equivalent to ``binary_string[0] is currently unused``.  Is this what you mean?